docs/troubleshooting.md

```diff
@@ -176,13 +176,14 @@ using the authentication token of the `antctl` ServiceAccount:
 
 ```bash
 # Get the token value of antctl account.
-TOKEN=$(kubectl get secrets -n kube-system -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='antctl')].data.token}"|base64 --decode)
+TOKEN=$(kubectl get secret/antctl-service-account-token -n kube-system -o jsonpath="{.data.token}"|base64 --decode)
```

Comment:
this will not work for old versions of Antrea. The alternative is:
```
kubectl get secrets -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='sa-test')].data.token}" | awk '{print $1}' | base64 --decode
```
but that requires awk

Unfortunately K8s jsonpath does not support selecting the first element after applying a filter expression. If we don't use awk, we may get 2 tokens for K8s versions before v1.24 (auto-generated secret, plus `secret/antctl-service-account-token`), which will not work.